ui/recieve_page.go

```diff
@@ -15,194 +23,242 @@ import (
 
 const PageReceive = "Receive"
 
-type receivePage struct {
-	pageContainer layout.List
+type walletAccount struct {
+	selectAccount []*gesture.Click
+}
 
-	isNewAddr, isInfo bool
-	addrs             string
+type walletAccountWidget struct {
+	title                    decredmaterial.Label
+	walletAccountModal       *decredmaterial.Modal
+	walletsList              layout.List
+	accountsList             layout.List
+	walletAccount            walletAccount
+	isWalletAccountModalOpen bool
+	walletAccounts           map[int]walletAccount
+	fromAccountBtn           *widget.Clickable
+}
+type receivePage struct {
+	pageContainer                layout.List
+	isNewAddr, isInfo            bool
+	addrs                        string
+	newAddrBtn, minInfo, copyBtn decredmaterial.Button
+	infoBtn, moreBtn             decredmaterial.IconButton
+	card                         decredmaterial.Card
+	receiveAddressLabel          decredmaterial.Label
```

Comment:
try to avoid naming your variable after its type. Your variable name should describe its content not its type. receiveAddress is just fine here.